Rakefile

```diff
@@ -1,24 +1,65 @@
+# frozen_string_literal: true
+
 require 'rspec/core/rake_task'
+require 'rubocop/rake_task'
+RuboCop::RakeTask.new(:rubocop) do |task|
+  # Use Rubocop's Github Actions formatter
+  task.formatters << 'github' if ENV['GITHUB_ACTIONS'] == 'true'
+end
+
+def commit_if_changes(msg, *args)
+  return if `git status --porcelain`.empty?
+
+  system('git', 'commit', '-m', msg, *args)
+end
+
+namespace :rubocop do
```

Comment:
I'm not happy with a huge implementation of this in a Rakefile. If we decide to go this route it belongs in a separate gem IMHO. Otherwise we end up duplicating it many times.

Reply:
maybe `task.options = ['--display-cop-names', '--display-style-guide', '--extra-details']` isn't even required anymore